src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -353,7 +353,7 @@
           "evacuation collection set (comprised of both young and old "     \
           "regions) is also bounded by this parameter.  In percents of "    \
           "total (young-generation) heap size.")                            \
-          range(1,100)                                                      \
+          range(0,100)                                                      \
```

Comment:
If we allow this, somebody _will_ file a bug because `-XX:ShenandoahEvacReserve=0` will cause other modes to crash with divide by zero error.